lib/Models/Terria.ts

```diff
@@ -516,8 +520,14 @@ export default class Terria {
       this.mainViewer.baseMap = baseMapSearch.mappable;
     } else {
       console.error(
-        `Couldn't find a basemap for unique id ${persistedBaseMapId}`
+        `Couldn't find a basemap for unique id ${persistedBaseMapId}. Trying to load init base map.`
```

Comment:
For clarity, it might be good to rename this method as `loadPersistedOrInitBaseMap()`

Reply:
changed